### ISSUE TITLE
[Feature]: Disabling Tools In Production Site

### ISSUE THREAD

**Issue by tqooze** [thedevdojo/voyager#366]
Is there any way to permanently disable database tools and menu builder or at least to disabling it beside for localhost? I would love that feature as it may cause much trouble if we leave it on production site.

**Comment by handiwijoyo** [comment]
There are no options to disable it for now. But you can disable the permission to `browse_database` and all of `menus` permissions to a certain Role. That will hide the menu and access to it.

**Comment by akpotter** [comment]
Also ask for this feature.
Please add "Settings", "Tools: (Menu Builder / Database)", "Media" to Edit Roles Permission.
Only admin but not other manager can see this function is more secure!!

Thanks.

**Comment by akazorg** [comment]
If we add a role_id to the menu, we can have unique menus per role.

**Comment by Jaquedeveloper** [comment]
Where can I set only a specific role to see the Database and the Menu Builder? Programmatically, I can't find the menu... where is it located? Also, @handiwijoyo this solution didn't worked for me, because it hides the entire menu and I want to show the menu and hide only this items if is not the specific role. How can I make it?

**Comment by marktopper** [comment]
@Jaquedeveloper, please share the content of the `permissions`, `role_permissions` and `menu_items` tables.

**Comment by Jaquedeveloper** [comment]
@marktopper, I've tried to take the browser_admin permission but it wasn't possible because if I do that, it redirects the site to the index page after login. There's a index page, the /admin/login page and then the admin administration panel page (/admin) on my project. If I set the browser_admin role permission to false, or delete it, it redirects the user to the index page after login, as if he's not logged in... Maybe I need a way to create a menu and show this menu for a specific role. Is it possible? I've also noticed that if I set a user to be able to only see a table, the button to edit and delete it is still there... it shouldn't be there, right? I've found some layouts and I'm looking for the correspondent generated layout in order to change it's code... Here are the table's looks:
**menu_items:**
<img width="788" alt="screen shot 2016-12-14 at 16 07 58" src="https://cloud.githubusercontent.com/assets/2420271/21198687/666c9872-c227-11e6-9a0f-b3f4b26a9569.png">

**role_permissions:**
<img width="194" alt="screen shot 2016-12-14 at 18 05 06" src="https://cloud.githubusercontent.com/assets/2420271/21198803/e4144d42-c227-11e6-9fa0-3b2681aa2929.png">

**Permissions:**
<img width="593" alt="screen shot 2016-12-14 at 18 06 40" src="https://cloud.githubusercontent.com/assets/2420271/21198863/2d926792-c228-11e6-88d9-bd187a62419e.png">
